python/examples/ros2-noenv/reader.py

```diff
@@ -0,0 +1,14 @@
+import sys
+
+from mcap_ros2.reader import read_ros2_messages
+
+
+def main():
+    for msg in read_ros2_messages(sys.argv[1]):
```

Comment:
hmm, seems like we should've gone with the MessageEvent naming scheme here, and I find the `.ros_msg` property name weird, but I see that this is the same as the ros1 support library